### ISSUE TITLE
Not compatible with Hami-WebUI。

### ISSUE THREAD

**Issue by bright-ren** [Project-HAMi/volcano-vgpu-device-plugin#39]
Installed volcano-vgpu-device-plugin v1.9.4 and HAMi-WebUI 1.0.3 in k8s 1.28.x. HAMi-WebUI  cannot detect any GPU node. 
After I modified some Node Lable and Annos,  HAMi-WebUI  can get the GPU nodes, but can not show any devices. 
Seems Device fields number not compatible, volcano-vgpu-device-plugin reports 5, while HAMi-WebUI requires 7 fiels.

**Comment by archlitchi** [comment]
yes, currently hami-webui only work on hami-scheduler, since it get the metrics from hami-scheduler:31993 port, we will implement that feature in future version

**Comment by geniuslc11** [comment]
Does the Hami-webui work on volcano-vgpu-device-plugin v1.9.4 now?

**Comment by mesutoezdil** [comment]
Old PRs and issues are being closed because they are no longer relevant. Or it was closed because of the inactivity or PR/issues owners.
If you believe the issue is still relevant, please submit feedback.
